framework/src/main/java/org/checkerframework/framework/util/defaults/QualifierDefaults.java

```diff
@@ -484,6 +498,26 @@ private void applyDefaults(Tree tree, AnnotatedTypeMirror type) {
         applyToTypeVar = false;
     }
 
+    /**
+     * Applies default annotations to {@code type}.
```

Comment:
This documentation is rather vague and general, but the method name is more evocative.  Which one should be changed?